File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    int length = s.length();
    string answer = "";
    
    if (length % 2 == 1) answer = s[length/2];
    else {
        answer += s[length/2-1];
        answer += s[length/2];
    }    
    return answer;
}

File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.js
function solution(s) {
    var answer = '';
    
    if (s.length % 2 == 1) {
        answer = s[~~(s.length/2)];
    }
    else {
        answer += s[s.length/2 - 1];
        answer += s[s.length/2];
    }
    
    return answer;
}